app/utilityFunctionsMostlyText.js

```diff
@@ -217,7 +217,7 @@ module.exports = {
         return result;
     },
     //moves them out of temp storage, creates image documents for them in the database, and returns arrays with their horizontality/verticality
-    //the non-first arguments are just stored in the image documents in the database. postType is "user" or "community"
+    //the non-first arguments are just stored in the image documents in the database. postType is "original" or "community"
```

Comment:
ah. the images have been being saved with either "user" or "community" in the context field, and i was thinking that posts were the same. note that for the image retrieval api function to keep working a further change has to be made here so that images are still either "user" or "community" and not "original"